packages/pretty-format/src/index.js

```diff
@@ -377,30 +377,33 @@ function printPlugin(
   depth: number,
   refs: Refs,
 ): string {
-  function boundPrint(val) {
-    return print(val, config, indentation, depth, refs);
-  }
-
-  function boundIndent(str) {
-    const indentationNext = indentation + config.indent;
-    return (
-      indentationNext + str.replace(NEWLINE_REGEXP, '\n' + indentationNext)
-    );
-  }
-
```

Comment:
Yes, thank you for asking. To avoid the problem in #3962 where `printPlugin` made unused closures for the callback functions, especially now that it might call either `serialize` or `print` property.